apps/frontend/src/libs/components/header/libs/components/user-popover/styles.module.css

```diff
@@ -0,0 +1,55 @@
+.user-popover {
+	max-width: 250px;
+}
+
+.user-info {
+	padding: 12px 16px;
+	border-bottom: 1px solid var(--color-border-secondary);
+}
+
+.user-name {
+	margin: 0;
+	font-size: 16px;
+	font-weight: 500;
+	line-height: 1.2;
+	color: var(--color-text-primary);
+	text-align: left;
+}
+
+.user-email {
+	margin: 0;
+	font-size: 14px;
+	font-weight: 500;
+	line-height: 1.3;
+	color: var(--color-text-secondary);
+	text-align: left;
+}
+
+.buttons {
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
+	padding: 4px;
+}
+
+.button {
+	width: 100%;
+	padding: 8px 12px;
+	font-size: 16px;
+	font-weight: 500;
+	line-height: 1.2;
+	color: var(--color-text-primary);
+	text-align: left;
+	cursor: pointer;
+	background-color: transparent;
+	border: none;
+	border-radius: 4px;
+}
+
+.button:hover {
+	background-color: color-mix(
+		in srgb,
+		var(--color-brand-primary-hover),
```

Comment:
This is for hover effect. I didn't find anything about hover effect of this elements in design, however I thought that it would be good to have one, so made the same as in select component.

Reply:
Let's use the same effect as for icon buttons / options menu on hover. In figma it's called background hover

Reply:
Okay